# Request
int8 CONST=1
string string_value
std_msgs/Float32 std_msgs_float
example_package/ExampleMsg example_msg

---

# Response
uint32 CONST=123456
uint32 int_value
example_package/ComposedMsg composed_msg
bool success
